presto-docs/src/main/sphinx/connector/prometheus.rst

```diff
@@ -0,0 +1,69 @@
+====================
+Prometheus Connector
+====================
+
+The Prometheus connector allows reading
+`Prometheus <https://prometheus.io/>`_.
+metrics as tables in Presto.
+
+Configuration
+-------------
+
+Create ``etc/catalog/prometheus.properties``
+to mount the Prometheus connector as the ``prometheus`` catalog,
+replacing the properties as appropriate:
+
+.. code-block:: none
+
+    connector.name=prometheus
+    prometheus-uri=http://localhost:9090
+    query-chunk-size-duration=1d
+    max-query-range-duration=21d
+    cache-duration=30s
+    bearer-token-file=/path/to/bearer/token/file
+
+Configuration Properties
+------------------------
+
+The following configuration properties are available:
+
+=================================== =====================================================================
+Property Name                       Description
+=================================== =====================================================================
+``prometheus-uri``                  Where to find Prometheus coordinator host
+``query-chunk-size-duration``       The duration of each query to Prometheus
```

Comment:
Added description of how connector works, how the translation from Presto query to Prometheus HTTP API call is done, so that config item descriptions later on have context. Done in c397641.